tests/src/application/validation/validators/email-validation.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { InvalidParamError } from '@/application/errors'
+import { EmailValidation, type EmailValidator } from '@/application/validation'
+
+const makeEmailValidatorStub = (): EmailValidator => {
+  class EmailValidatorStub implements EmailValidator {
+    isValid (email: string): boolean {
+      return true
+    }
+  }
+  return new EmailValidatorStub()
+}
+
+interface SutTypes {
+  sut: EmailValidation
+  emailValidatorStub: EmailValidator
+}
+
+const makeSut = (): SutTypes => {
+  const fakeFieldName: string = 'fieldName'
+  const emailValidatorStub = makeEmailValidatorStub()
+  const sut = new EmailValidation(fakeFieldName, emailValidatorStub)
+  return {
+    sut,
+    emailValidatorStub
+  }
+}
+
+describe('EmailValidation', () => {
+  it('should returns InvalidParamError if fieldName is not a string', () => {
+    const { sut } = makeSut()
+
+    const result = sut.validate({ fieldName: 11 })
```

Comment:
to be more descriptive you can use an name for this value 11. Like `notStringValue = 11`